packages/compile-solidity/src/profiler/loadParser.ts

```diff
@@ -46,8 +53,8 @@ async function loadParser({ events, compilers: { solc: solcConfig } }) {
   return makeParseImports(parserSolc);
 }
 
-function makeParseImports(parser) {
-  const parseImports = body => {
+function makeParseImports(parser: any) {
```

Comment:
Is there not a better type for this? If not, maybe this should be a generic?

Reply:
If there really is no better type for this apart from `any`, or if defining a valid type constraint as part of this PR would be high effort, you might also consider defining a `SolidityParser` type alias that is still `any`, as that way it'll be easy to update in the future should better types become available.